src/Scripting/Core/Hosting/ObjectFormatter/ObjectFormatter.cs

```diff
@@ -12,6 +12,7 @@
 
 namespace Microsoft.CodeAnalysis.Scripting.Hosting
 {
+    using System.Globalization;
```

Comment:
Can you please move this out of the namespace declaration?  The alias is only in here to resolve an ambiguity (I assume).